.changesets/allow-pino-transport-to-be-used-as-a-transport.md

```diff
@@ -0,0 +1,50 @@
+---
+bump: patch
+type: fix
+---
+
+Allow Pino transport to be used as a transport. Before, our Pino transport could only be used as a destination:
+
+```js
+import pino from "pino";
+import { Appsignal, AppsignalPinoTransport } from "@appsignal/nodejs";
+
+pino(AppsignalPinoTransport({
+  client: Appsignal.client,
+  group: "pino"
+}));
+```
+
+This meant it was not possible to log both to our transport and to another destination.
+
+Now, it is also possible to use it as a Pino transport, with the `transport` Pino config option or the `pino.transport()` function:
+
+```js
+import pino from "pino";
+
+pino({
+  transport: {
+    target: "@appsignal/nodejs/pino",
+    options: {
+      group: "pino"
```

Comment:
```suggestion
      group: "application"
```

The test app uses this. If it's the log group, we use "application" for app logs.

Reply:
Yes, it's the log group. This is just an example value in the changeset snippet -- the customer can set it to whatever value they want.